features/steps/assign_incomplete_case.py

```diff
@@ -0,0 +1,76 @@
+from behave import *
+from selenium.webdriver.common.action_chains import ActionChains
+from create_eligible_finances import create_eligible_finance
+
+
+@given(u'case notes are empty')
+def step_impl(context):
+    notes = context.helperfunc.find_by_name('case.notes')
+    assert len(notes.text) == 0
+
+
+@step(u'I have created a user')
+def step_impl(context):
+    context.execute_steps(u'''
+        When I select 'Create new user'
+        And enter the client's personal details
+        And I click the save button on the screen
+    ''')
+
+
+@given(u'I have created a valid discrimination scope')
```

Comment:
Remember [we had a lil talk here](https://mojdt.slack.com/archives/CFUESB43G/p1635948246061400?thread_ts=1635945901.060200&cid=CFUESB43G) about using this step in my test to replace the `I am on the Finances tab with the ‘Details’ sub-tab preselected`